src/components/hover-cards.vue

```diff
@@ -139,14 +139,12 @@ const hide = () => {
   component.value = null;
   placement.value = undefined;
 };
-watch(() => hoverCard.anchor, (anchor) => {
-  /* If hoverCard.state is `true`, then there are two possibilities. It could be
-  that hoverCard.state has changed from `false` to `true`. Or it could be that
-  it was `true` already, but hoverCard.anchor has changed. (That would be pretty
-  unusual, but it could happen if preventHide is set to `true` in
-  useHoverCard().) But most importantly, if hoverCard.state is `true`, then we
-  need to hide the current hover card before we show a new one. */
-  if (hoverCard.state) hide();
```

Comment:
This `if` meant that the `hide()` function was not called after `hoverCard.state` changed to `false`.